recipes/server.rb

```diff
@@ -66,3 +66,13 @@
 sysctl_param 'vm.swappiness' do
   value 0
 end
+
+cookbook_file '/usr/local/bin/mysql-accounting' do
+  source 'mysql-accounting'
+  mode '0755'
+end
+
+cron 'mysql-accounting' do
+  command '/usr/local/bin/mysql-accounting'
+  time :daily
+end
```

Comment:
Would be nice to have a test for this cron job. I believe this also depends on the cron cookbook.